publiccode.yml

```diff
@@ -21,113 +15,67 @@ outputTypes:
 platforms:
   - GNU/Linux
 
-# For a list of allowed values, check out w3id.org/publiccode/version/0.1/tags.html
-#
-# Only stick to these tags! If you need additional tags (localised),
-# you have the chance of adding them under the description/freeTags key
-tags:
-  - concorsi personale tecnico amministrativo e docenti
-  - bandi
+categories:
+  - procurement
+  - hr
 
 usedBy:
   - Università della Calabria
 
-# roadmap: "https://github.com/italia/django-peo/roadmap"
-
 developmentStatus: stable
-softwareType: "standalone"
-
+softwareType: standalone/web
 
 description:
-  eng:
+  en:
     localisedName: Peo
     genericName: Django peo
-    shortDescription: >
+    shortDescription: |- 
           Gestione, redazione e pubblicazione Bandi e Concorsi per
           personale Tecnico Amministrativo e personale Docente.
 
-    longDescription: >
+    longDescription: |- 
```

Comment:
This key does not pass validation. The standard mandates to have a 
`Tipo: stringa (min 500 caratteri, max 10000 caratteri)` so some more chars are needed.